modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_state.yml

```diff
@@ -4,6 +4,8 @@ dependencies:
   config:
     - field.storage.node.field_location_state
     - node.type.branch
+  module:
+    - datalayer
```

Comment:
can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done